tutorials/pretrain_openwebtext.md

```diff
@@ -0,0 +1,139 @@
+# Pretrain Llama 2 on OpenWebText
+
+This tutorial will walk you through setting up the OpenWebText dataset and launching the pretraining script.
+
+## What's OpenWebText
+
+[OpenWebText](https://github.com/jcpeterson/openwebtext) is an open-source reproduction of OpenAI's unreleased WebText training dataset, which was originally used to train GPT-2. 
```

Comment:
Nit: Might be odd that each sentence is its own paragraph?

Reply:
Agreed